spec/fake_signalflow/ssl.rb

```diff
@@ -0,0 +1,60 @@
+require 'openssl'
+require 'tempfile'
+
+class SelfSignedCertificate
```

Comment:
I don't think you'd need this if your tests connected to a `ws://` fake endpoint.

Reply:
True, but I meant to test peer verification with this and forgot so I'll add that now to the tests.  It is pretty important to ensure in tests that the client does verification since the client sends auth tokens.